stdlib/public/core/KeyPath.swift

```diff
@@ -3450,6 +3425,12 @@ internal struct InstantiateKeyPathBuffer: KeyPathPatternVisitor {
                                      mutable: Bool,
                                      offset: KeyPathPatternStoredOffset) {
     let previous = updatePreviousComponentAddr()
+    switch kind {
+        case .struct:
```

Comment:
I think autoformatting nudged these when they were copy-pasted (here and below)

```suggestion
    case .struct:
```